src/server/common/model/answer/radio-button/radio-button.test.js

```diff
@@ -168,14 +169,39 @@ describe('RadioButton', () => {
     }
 
     it('should return everything (except errors) to render in the template', () => {
-      expect(answer.viewModel({ validate: false })).toEqual(defaultViewModel)
+      expect(invalidAnswer.viewModel({ validate: false })).toEqual(
+        defaultViewModel
+      )
     })
 
     it('should return everything (including errors) to render in the template', () => {
-      expect(answer.viewModel({ validate: true })).toEqual({
+      expect(invalidAnswer.viewModel({ validate: true })).toEqual({
         ...defaultViewModel,
         errorMessage: { text: 'Select an option' }
       })
     })
+
+    describe('radio button layout', () => {
+      const originalLayout = testRadioConfig.layout
+
+      afterEach(() => {
+        testRadioConfig.layout = originalLayout
+      })
```

Comment:
I think it's much clearer here if we declare a new class - rather than the user having to keep track of how the RadioButtonTest config has been mutated.

With the double-configs, I can see why this pattern is tempting, but I'd rather:
-  port us over to [this pattern](https://github.com/DEFRA/apha-apps-perms-move-animal-ui/pull/171/files#diff-ee1323f5f98f8878ed19567c758f80fbed2dcd4504e6cd2214f6cfb4e45a1c0dR58-R69) for RadioButton 
- set up testing variations, e.g. [like this](https://github.com/DEFRA/apha-apps-perms-move-animal-ui/pull/171/files#diff-14aa05114e7d09fc8ba7bd396d47afa3f1400eba67640cf3fbbdfcaaa8f9e3bcR17-R22)

Would you mind to do this as part of this ticket?  At the end of DSFAAP-799, I'll make sure all of the models are using this simpler config / static config set up.